src/Surfnet/StepupRa/RaBundle/Form/Type/VerifyIdentityType.php

```diff
@@ -21,18 +21,22 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\Length;
 
 class VerifyIdentityType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder->add('documentNumber', 'text', [
             'label' => 'ra.form.verify_identity.document_number.label',
+            'constraints' => new Length(6),
```

Comment:
0. Is there a specific reason you added this here and not on the `VerifyIdentityCommand`? If it's an oversight, I recommend adding it there :wink:
0. Did you get confirmation on this length-must-be-exactly-6? Maybe some foreign document has 6 or less characters (I don't know), and then the idea is that not the entire number is entered (afaik).